### DISCUSSION TITLE
Skip to last slide

### DISCUSSION THREAD

**Discussion by TulioPerez** [AppIntro/AppIntro#975] [Q&A] [answered]
Sometimes we want users to arrive at the last slide when they press "skip." I've been trying to find a way to do this  without calling "goToNextSlide(false)" repeatedly until arriving at the final slide. How can this be done correctly?

Also, how can one left-justify the description text on the slide layouts without recreating and setting a new layout file?

**Comment by cortinico** [comment, marked as answer]
> Sometimes we want users to arrive at the last slide when they press "skip." I've been trying to find a way to do this without calling "goToNextSlide(false)" repeatedly until arriving at the final slide. How can this be done correctly?

This is not easily possible. The reason is that the library handles the permission management when transitioning between slides. If we allow to "skip" to a certain slide, handling permissions would have to be adjusted. Is not impossible but would have to be implemented.

> Also, how can one left-justify the description text on the slide layouts without recreating and setting a new layout file?

You can declare a new style called `AppIntroDefaultText` and override it there.

**Reply by TulioPerez** [reply]
Thank you for your help and prompt response - AppIntro is a great help btw!
